.gitignore

```diff
@@ -1,85 +1,16 @@
-# Built application files
```

Comment:
@magda2307 Please try to use predefined .gitignore file. We won't files from .idea folder